library/psa_crypto.c

```diff
@@ -1260,6 +1260,137 @@ static psa_status_t psa_validate_optional_attributes(
     return PSA_SUCCESS;
 }
 
+static psa_status_t psa_validate_ecc_key_attr(const psa_key_attributes_t *attributes)
+{
+    psa_algorithm_t alg = psa_get_key_algorithm(attributes);
+    size_t key_bits_attr = psa_get_key_bits(attributes);
+    psa_key_type_t type = psa_get_key_type(attributes);
+
+    /* With algorithm we can't evaluate more fields */
+    if (alg == PSA_ALG_NONE) {
+        return PSA_SUCCESS;
+    }
+
+    /* It is not mandatory to set the key bits field, so zero is valid*/
+    if (key_bits_attr == 0) {
+        return PSA_SUCCESS;
+    }
+
+    /* Check if the size matches the curve family */
+    switch (PSA_KEY_TYPE_ECC_GET_FAMILY(type)) {
+        case PSA_ECC_FAMILY_SECP_K1:
+            if (key_bits_attr != 192 && key_bits_attr != 225 && key_bits_attr != 256) {
```

Comment:
225 => 224

Reply:
According to spec it has 225 bits 
https://arm-software.github.io/psa-api/crypto/1.1/api/keys/types.html#c.PSA_ECC_FAMILY_SECP_K1

Reply:
ah, okay